consent-ws/src/main/java/org/broadinstitute/consent/http/service/ontologyIndexer/ElasticSearchSupport.java

```diff
@@ -0,0 +1,39 @@
+package org.broadinstitute.consent.http.service.ontologyIndexer;
+
+import org.apache.http.Header;
+import org.apache.http.HttpHost;
+import org.apache.http.message.BasicHeader;
+import org.broadinstitute.consent.http.configurations.ElasticSearchConfiguration;
+import org.elasticsearch.client.RestClient;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.util.stream.Collectors;
+
+@SuppressWarnings("WeakerAccess")
+public class ElasticSearchSupport {
+
+    public static RestClient getRestClient(ElasticSearchConfiguration configuration) {
+        HttpHost[] hosts = configuration.
+            getServers().
+            stream().
+            map(server -> new HttpHost(server, 9200, "http")).
+            collect(Collectors.toList()).toArray(new HttpHost[configuration.getServers().size()]);
```

Comment:
not sure if this applies, but should we be using https? would that change this code?